### PR TITLE
Add commas to price formatting for G-Cloud services

### PR DESCRIPTION
**Pull request by alex9smith**
Add commas to the prices for G-cloud services so they're easier to read, and bump the minor version number

Addresses https://trello.com/c/UL5WzBD7/1412-add-commas-to-price-formatting-on-g-cloud-service-pages

### CODE REVIEW

File: dmcontent/formats.py [hunk 1]
```diff
@@ -2,16 +2,40 @@
...
+    Format a number with commas if it's greater than 999.
+    eg: `1000 -> '1,000'
+        123456.78 -> '123,456.78'
+        '1000000' -> '1,000,000'
+        500 -> '500'
+
+    :param number: The number to be formatted. Either an int, a float or a str.
+    :return: A string of the comma formatted number
+    """
+    if type(number) == str:
```
Comment (bjgill) [inline dmcontent/formats.py:16]: Is it necessary to provide exactly `str`? Could we use [`isinstance`] to allow for subclasses of `str`?

(and ditto for `int` below)

**Review by bjgill — COMMENTED** [review]
Looks sensible - I particularly like the test.

Just one question about the type testing and whether we can be more permissive.

File: dmcontent/formats.py (hunk 1 above)
Comment (alex9smith) [reply dmcontent/formats.py:16]: That does seem sensible. I'll have a look at where this gets called to check we're not doing anything unusual with types there but if not I'll update

**Review by bjgill — APPROVED** [review]
:shipit:

**Pull request merged**